### PR TITLE
Add config map to install custom CA certs

### PR DESCRIPTION
**Pull request by olliewalsh**
Required for TLS public endpoints where we cannot rely on IPA client to
install CA certs.

### CODE REVIEW

File: pkg/common/configmap.go [hunk 1]
```diff
@@ -202,3 +204,39 @@ func GetConfigMapAndHashWithName(r ReconcilerCommon, configMapName string, names
...
+	err := r.GetClient().Get(context.TODO(), types.NamespacedName{Name: configMapName, Namespace: object.GetNamespace()}, configMap)
+	if err != nil {
+		if k8s_errors.IsNotFound(err) {
+			cond.Message = fmt.Sprintf("%s config map does not exist: %v", configMapName, err)
+			cond.Reason = ospdirectorv1beta1.ConditionReason(conditionDetails.ConditionNotFoundReason)
+			cond.Type = ospdirectorv1beta1.ConditionType(conditionDetails.ConditionNotFoundType)
+
+			LogForObject(r, cond.Message, object)
+
+			return configMap, ctrl.Result{}, err
```
Comment (stuggi) [inline pkg/common/configmap.go:231]: From description we want to reconcile when not found, so should be:

`return configMap, ctrl.Result{RequeueAfter: time.Duration(requeueTimeout) * time.Second}, nil`

**Review by stuggi — COMMENTED** [review]
looks good, just one small change needed as the return from GetConfigMap is missing the timeout to reconcile.

File: pkg/common/configmap.go (hunk 1 above)
Comment (olliewalsh) [reply pkg/common/configmap.go:231]: yes, well spotted!

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**